src/app/modules/user/user.service.ts

```diff
@@ -166,27 +166,32 @@ export const updateUserContact = (
  * Updates the user document with the userId with the given latest seen feature update date and
  * returns the populated updated user.
  * @param userId the user id of the user document to update
- * @param latestLastSeenFeatureUpdateDate the latest last seen feature update date to update with
+ * @param date the latest last seen feature update date to update with
  * @returns ok(true) if update was successful
  * @returns err(MissingUserError) if user document cannot be found
  * @returns err(DatabaseError) if any error occurs whilst querying the database
  */
 export const updateUserLastSeenFeatureUpdateDate = (
   userId: IUserSchema['_id'],
-  latestLastSeenFeatureUpdateDate: Date,
-): ResultAsync<true, MissingUserError | DatabaseError> => {
+  date: Date,
```

Comment:
instead of just passing in a date, should we just call `Date.now()` in this function?

Reply:
Yeap I think I should have done that earlier.